ui/src/app/shared/conditions.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+import {WarningWorkflowConditions, WorkflowCondition} from '../../models';
+
+interface Props {
+    conditions: WorkflowCondition[];
+}
+
+export class Conditions extends React.Component<Props> {
+    public render() {
+        return (
+            <>
+                {this.props.conditions &&
+                    Object.entries(this.props.conditions).map(([_, condition]) => {
+                        return (
+                            <div key={condition.type} style={{lineHeight: '120%', marginTop: '16px'}}>
```

Comment:
not sure about these style? why needed?

Reply:
The different rows in the Workflow Details panel are spaced with a `line-height: 50px` which all of its children inherit. This would result in this:

<img width="1028" alt="image" src="https://user-images.githubusercontent.com/614838/77949707-43f71980-727c-11ea-88ee-62c221eb5922.png">

This style overwrites the inheritance to make the text look normal.